### PR TITLE
added support for lpar start command

### PR DESCRIPTION
**Pull request by saiprasanna-16**
added missing support for lpar start group command

### CODE REVIEW

**Comment by andy-maier** [comment]
@saiprasanna-16 
PR #545 fixes the safety issues that cause the failures for this PR.

Please rebase the PR to latest master, resolve the merge conflict, and then it should pass the test.

File: zhmccli/_cmd_lpar.py [hunk 1]
```diff
@@ -519,6 +519,28 @@ def lpar_console(cmd_ctx, cpc, lpar, **options):
     cmd_ctx.execute_cmd(lambda: cmd_lpar_console(cmd_ctx, cpc, lpar, options))
 
 
+@lpar_group.command('start', options_metavar=COMMAND_OPTIONS_METAVAR)
+@click.argument('CPC', type=str, metavar='CPC')
+@click.argument('LPAR', type=str, metavar='LPAR')
+@click.option('-y', '--yes', is_flag=True, callback=abort_if_false,
+              expose_value=False,
+              help='Skip prompt to confirm starting of the LPAR.',
+              prompt='Are you sure you want to start the LPAR ?')
```
Comment (andy-maier) [inline zhmccli/_cmd_lpar.py:528]: I don't think we should have an extra confirmation for starting the LPAR. We only should have such prompts on actions that cause a disruption to something that runs (i.e. on stop).

File: zhmccli/_cmd_lpar.py [hunk 2]
```diff
@@ -519,6 +519,28 @@ def lpar_console(cmd_ctx, cpc, lpar, **options):
...
+@click.option('--allow-status-exceptions', is_flag=True, required=False,
+              help='Allow status "exceptions" as a valid end status.')
+@add_options(ASYNC_TIMEOUT_OPTIONS)
+@click.pass_obj
+def lpar_start(cmd_ctx, cpc, lpar, **options):
+    """
+    Start the processors from processing instructions of an LPAR.
```
Comment (andy-maier) [inline zhmccli/_cmd_lpar.py:535]: Grammar: Change to "Start the processors for processing instructions of an LPAR."

**Review by andy-maier — CHANGES_REQUESTED** [review]

File: zhmccli/_cmd_lpar.py (hunk 1 above)
Comment (saiprasanna-16) [reply zhmccli/_cmd_lpar.py]: Okay

File: zhmccli/_cmd_lpar.py (hunk 2 above)
Comment (saiprasanna-16) [reply zhmccli/_cmd_lpar.py]: Done

**Review by andy-maier — APPROVED** [review]

**Pull request merged**